CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ set(REMILL_BUILD_SEMANTICS_DIR_SPARC64 "${CMAKE_CURRENT_BINARY_DIR}/lib/Arch/SPA
 set(REMILL_BUILD_SEMANTICS_DIR_PPC64_32ADDR "${CMAKE_CURRENT_BINARY_DIR}/lib/Arch/PPC/Runtime")
 set(REMILL_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/include")
 set(REMILL_LIB_DIR "${CMAKE_CURRENT_SOURCE_DIR}/lib")
-
+set(FETCHCONTENT_BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/deps)
```

Comment:
<img width="852" height="396" alt="image" src="https://github.com/user-attachments/assets/1fe73838-e377-4875-b503-d3ab2ade62f3" />

If you set it, use a CACHE variable (with the original default value) so it becomes visible in the CMake GUI and users can edit it. If you want to put it in a `deps` folder here, at the very least it should be added to `.gitignore` as well.